### DISCUSSION TITLE
Resolved

### DISCUSSION THREAD

**Discussion by syntaxerror41** [Mikubill/sd-webui-controlnet#1212] [Ideas]
Resolved

**Comment by lbeltrame** [comment]
A workaround is to use inpaint with ControlNet. In my limited testing, it works (I picked a part of the image that I wanted redone, used canny, and I used inpaint with reasonable weight, and it improved a great deal).

**Comment by Chubbly** [comment]
Is this not solved using Adetailer https://github.com/Bing-su/adetailer.git?

It allows you to use ControlNets too.

**Reply by dsynclabsllc** [reply]
I was looking to do this as well. This would be super helpful when it comes to posing.

**Comment by Rkkss** [comment]
Can webui even allow you to do it?
feel like you need to combine controlNET with segment anything.
